tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -95,7 +95,10 @@ Define {
 	BT_PB_DAI_PIPELINE_SRC "copier.host.9.1"
 	BT_PB_PIPELINE_STREAM_NAME "dai-copier.SSP.10.1"
 	GOOGLE_RTC_AEC_SUPPORT		0
+	# assign AEC LL pipelines to secondary core
 	GOOGLE_AEC_CORE_ID		0
+	# assign RTC_AE DP task to secondary core
+	GOOGLE_AEC_DP_CORE_ID		0
```

Comment:
Hmm, @yongzhi1 , I don't understand the comment. "assign to secondary core" but code sets CORE_ID to 0...?

Reply:
Ack, maybe better to say "override in tplg-targets.cmake to run on secondary core"?